package.json

```diff
@@ -123,6 +123,10 @@
               ],
               "default": "off",
               "description": "Traces the communication between VS Code and the language server."
+            },
+            "version": {
```

Comment:
Nitpick: Can we call this `required_version`? Just to have it aligned with the naming convention within Terraform and to communicate more explicitly that a constraint is expected (rather than just a version) https://www.terraform.io/docs/language/settings/index.html#specifying-a-required-terraform-version

Reply:
Sure that can be done.

Reply:
Done.